scripts/init-project.sh

```diff
@@ -18,6 +18,9 @@ docker-compose exec web drush en contentpool_client -y
 # see drunomics/contentpool:scripts/init-project.sh
 docker-compose exec web drush contentpool-client:setup http://replicator:changeme@contentpool-project.localdev.space/relaxed
 
+# Override config from data model.
+docker-compose exec web drush config-import --partial --source=modules/drunomics/contentpool_data_model/config/override -y
```

Comment:
Let's do this in the installation routine of contentpool_data_model with custom code. This is necessary for every contentpool_data_model installation - not only tests, so we'll need to take care of it so that usual installations work as well.